### PR TITLE
Updated license and copyright years

### PR DESCRIPTION
**Pull request by lizgehret**
Updated license and copyright header years to include 2021

### CODE REVIEW

**Comment by lizgehret** [comment]
Ran flake8 and made necessary updates.

**Review by thermokarst — CHANGES_REQUESTED** [review]
Please see my hashpling comments over in https://github.com/qiime2/busywork/pull/125#pullrequestreview-652846448, thanks!

**Review by lizgehret — COMMENTED** [review]
Moving hashpling to top of script

**Review by lizgehret — COMMENTED** [review]
Removing line breaks after hashplings

File: bin/email_sheets.py [hunk 1]
```diff
@@ -1,4 +1,11 @@
-#!/usr/bin/env python
+# !/usr/bin/env python
```
Comment (thermokarst) [inline bin/email_sheets.py:1]: Minor typo, there is an extra space tucked away in here that should be removed.

```suggestion
#!/usr/bin/env python
```

File: bin/generate_accts.py [hunk 2]
```diff
@@ -1,4 +1,11 @@
-#!/usr/bin/env python
+# !/usr/bin/env python
```
Comment (thermokarst) [inline bin/generate_accts.py:1]: Same comment as above!

**Review by thermokarst — CHANGES_REQUESTED** [review]

**Review by lizgehret — COMMENTED** [review]
Removed space typos within hashpling

**Review by thermokarst — APPROVED** [review]
🦖

**Pull request merged**